Description: move to @rollup/plugin-typescript
 move from @wessberg/rollup-plugin-ts to @rollup/plugin-typescript
Author: Ayoyimika Ajibade <ayoyimikaajibade@gmail.com>
Forwarded: not-needed
Last-Update: 2021-11-04
--- a/package.json
+++ b/package.json
@@ -57,7 +57,7 @@
     "@types/node": "^14.0.27",
     "@typescript-eslint/eslint-plugin": "^4.0.0",
     "@typescript-eslint/parser": "^4.0.0",
-    "@wessberg/rollup-plugin-ts": "^1.3.2",
+    "@rollup/plugin-typescript": "^6.0.0",
     "c8": "^7.3.0",
     "chai": "^4.2.0",
     "chalk": "^4.1.0",
--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,8 +1,9 @@
-import ts from '@wessberg/rollup-plugin-ts'
+import ts from '@rollup/plugin-typescript';
 
 const output = {
+  dir: "build",
   format: 'cjs',
-  file: './build/index.cjs',
+  entryFileNames: 'index.cjs',
   exports: 'default'
 }
 
--- a/tsconfig.json
+++ b/tsconfig.json
@@ -4,14 +4,12 @@
     "outDir": "build",
     "rootDir": ".",
     "sourceMap": false,
+    "lib": ["es2019"],
     "target": "es2017",
     "moduleResolution": "node",
     "module": "es2015"
-  },  
+  },
   "include": [
     "lib/**/*.ts"
-  ],
-  "exclude": [
-    "lib/cjs.ts"
   ]
-}
\ No newline at end of file
+}
